development/kops/cluster_wait.sh

```diff
@@ -13,8 +13,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -exo pipefail
+set -eo pipefail
 
-kops validate cluster --wait 10m
-kubectl apply -f ./aws-iam-authenticator.yaml
-kubectl delete pod -n kube-system -l k8s-app=aws-iam-authenticator
+#
+# Add IAM configmap
+COUNT=0
+echo 'Waiting for cluster to come up...'
+while ! kubectl apply -f ./${KOPS_CLUSTER_NAME}/aws-iam-authenticator.yaml
+do
+    sleep 5
+    COUNT=$(expr $COUNT + 1)
+    if [ $COUNT -gt 120 ]
+    then
+        echo "Failed to configure IAM"
+        exit 1
+    fi
+    echo 'Waiting for cluster to come up...'
+done
+
+set -x
+kops validate cluster --wait 3m
+
+# This may not be required
+# kubectl delete pod -n kube-system -l k8s-app=aws-iam-authenticator
```

Comment:
So is this actually required? If we are leaving it as a reference, let's explain that a bit more in the comment above

Reply:
It doesn't seem to be needed. Conformance tests pass without a restart. Old comment I guess.

Reply:
Removed that comment.